frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -1664,6 +1664,7 @@ ACTION_TYPE_CANNOT_CHANGE_INITIAL_RUN_DATA=Cannot ${action} ${type}. The Initial
 ACTION_TYPE_CANNOT_SET_MANUAL_PINNING=Cannot ${action} ${type}. The CPU Pinning policy is set to 'Manual' but no cpu pinning string is provided.
 ACTION_TYPE_CANNOT_RESIZE_AND_PIN_SINGLE_CORE=Cannot ${action} ${type}. The CPU Pinning policy 'Resize and Pin NUMA' cannot be applied to a VM that is pinned to a single core host.
 ACTION_TYPE_CANNOT_RESIZE_AND_PIN_AND_NUMA_SET=Cannot ${action} ${type}. The VM NUMA configuration cannot be applied when using the CPU Pinning policy 'Resize and Pin NUMA'.
+ACTION_TYPE_CANNOT_SET_DEDICATED_PINNING_WITH_NUMA_NODES_PINNED=Cannot ${action} ${type}. The VM NUMA nodes cannot be pinned when using the CPU Pinning policy 'Dedicated'.
```

Comment:
cannot be **manually** pinned?

Reply:
can it be automatically pinned? do we keep the numa pinning after the vm is powered off?

Reply:
This is static configuration - so yes. Unless we will decide to clear it (would be also good for resize and pin) on shutdown.
Currently, it stays as is, on migration we should change it in the DB on successful migration.
@ahadas note it will difficult as CPU pinning string in terms of engine recovery (another field to know the future pinning of the numa).

Reply:
> This is static configuration - so yes. Unless we will decide to clear it (would be also good for resize and pin) on shutdown. 

that's not a good reason - to keep it because we didn't have another field
for a VM with resize-and-pin policy, the pinning should be set/overridden when running the VM so keeping it when it's down is not that great

> @ahadas note it will difficult as CPU pinning string in terms of engine recovery (another field to know the future pinning of the numa).

can you rephrase?

Reply:
> > @ahadas note it will difficult as CPU pinning string in terms of engine recovery (another field to know the future pinning of the numa).
> 
> can you rephrase?

Sure, for the migrations i need to add temporary CPU pinning string, just for the flow where the engine recovers. For NUMA it will be required as well, for the exact same flow. Unless you are saying: I don't care if the DB don't correlate with the actual QEMU process. When thinking on this a bit more - that should be fine. 
The CPU pinning string is really important to know the taken physical CPUs, but we don't have any usage for the NUMA details. It would not be nice, especially when admin can view the DB/API and think he has something but in reality it's configured with other values. That would be like that until the VM is down/migrated again.

Reply:
reading this again, I don't see why the thread diverged to whether or not we need to have temporary state persisted
the comment above was about the fact that users should not manually pin the NUMA nodes with dedicated CPUs because they would hopefully be automatically pinned when the NUMA count is specified with the dedicated policy